<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Post Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="main_layout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>324</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mail_layout">
        <item>
         <widget class="QLabel" name="from_label">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="title_label">
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Suject:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item>
         <widget class="QPushButton" name="download_attach">
          <property name="text">
           <string>download attachment</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMail">
    <property name="title">
     <string>Mail</string>
    </property>
    <addaction name="actionAdd_account"/>
    <addaction name="actionNew_message"/>
   </widget>
   <addaction name="menuMail"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_account">
   <property name="text">
    <string>Add mailbox</string>
   </property>
  </action>
  <action name="actionNew_message">
   <property name="text">
    <string>New message</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
